private.php

```diff
@@ -1112,6 +1112,7 @@
 
 	// Fetch recipient names from the database
 	$bcc_recipients = $to_recipients = array();
+	$bcc_form_val = $to_recipients = array();
```

Comment:
Why did you add it like that? Everything is an empty array, so you could just do `$bcc_form_val = $bcc_recipients = $to_recipients = array();`